variable_set_variable.go

```diff
@@ -49,7 +49,7 @@ type VariableSetVariable struct {
 	Sensitive   bool         `jsonapi:"attr,sensitive"`
 
 	// Relations
-	VariableSet *VariableSet `jsonapi:"relation,configurable"`
+	VariableSet *VariableSet `jsonapi:"relation,varset"`
```

Comment:
Are `varset.data.id` and `varset.data.type` being exposed?

<img width="532" alt="Screen Shot 2022-08-30 at 5 27 48 PM" src="https://user-images.githubusercontent.com/29918071/187546576-cb5708e7-a278-4e2e-85af-fb7228c4f96a.png">

Reply:
Yep! The relationship is automatically included in the response body.